<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NAIM" failures="1" tests="6" name="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" time="71.226" errors="0" timestamp="2023-09-26T04:18:37 EDT" skipped="2">
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC001" time="11.173">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC001(R1dot1_TS001_RegisterFunctionality.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- r1dot1_TS001_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC001" time="18.033">
    <skipped/>
  </testcase> <!-- r1dot1_TS001_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC001" time="10.561">
    <skipped/>
  </testcase> <!-- r1dot1_TS001_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC003" time="11.605"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC004" time="9.355"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC028" time="10.499"/>
</testsuite> <!-- com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality -->
